src/instigator.rs

```diff
@@ -34,7 +34,7 @@ pub const COMPONENT_RECORD_ANNOTATION: &str = "component_record_annotation";
 
 /// Type alias for the results that all instantiation operations return
 pub type InstigatorResult = Result<(), Error>;
-type OpResource = Object<ApplicationConfiguration, Status>;
+type OpResource = Object<ApplicationConfiguration, OAMStatus>;
```

Comment:
I also use OAMStatus instead of Status here.
Because Status is a wrapper of OAMStatus with option. kube::Object will add Option wrapper for it's status generic. So if we use Status here, we will finally have `option<option< OAMStatus >>` that makes annoyance.

Reply:
Ah! Yes, that is cleaner